### ISSUE TITLE
What Are The Campus Run Systems Used?

### ISSUE THREAD

**Issue by kinlane** [low-hanging-fruit/davidson-college#2]

**Comment by windley** [comment]
Kin, I presume this question is asking:
1. What SIS is Davidson running
2. What LMS is Davidson running
3. Is Davidson using a campus-wide advisement/case management tool? 
4. What HR/Payroll (ERP) is Davidson running
5. What other systems play a major role in campus operations (facilities, maintenance, events, etc.)

**Comment by kinlane** [comment]
Precisely. What are the big beasts already in place, and what are some of the lesser known ones.
